web_auto_refresh/web_auto_refresh.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2004-today OpenERP SA (<http://www.openerp.com>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+
+import openerp
+from openerp.models import BaseModel
+from openerp import models, api
+
+
+class MailNotification(models.Model):
+    _inherit = 'mail.notification'
+
+    @api.model
+    def create(self, vals):
+        res = super(MailNotification, self).create(vals)
+        bus = self.env['bus.bus']
+        user_obj = self.env['res.users']
+        users = user_obj.search([('partner_id', '=', res.partner_id.id)])
+        for user in users:
+            bus.sendone(self._name, user.id)
+        return res
+
+
+create_original = BaseModel.create
+
+
+@openerp.api.model
+@openerp.api.returns('self', lambda value: value.id)
+def create(self, vals):
+    record_id = create_original(self, vals)
+    auto_refresh_kanban_list(self)
+    return record_id
+BaseModel.create = create
+
+
+write_original = BaseModel.write
+
+
+@openerp.api.multi
+def write(self, vals):
+    result = write_original(self, vals)
+    auto_refresh_kanban_list(self)
+    return result
+BaseModel.write = write
```

Comment:
I know, this is a test, if we decide to use it we can change it to a server action.